src/main/java/org/automation/pageObjects/GiftCardConfigurationPopup.java

```diff
@@ -84,8 +85,23 @@ public String getMaxGiftCardToolTipMessage() {
 		return getText_custom(By.id(id));
 	}
 
+	public String getFundingSourceToolTipMessage() {
+		moveToWebElement(fundingSourceTbx);
+		String id=getAttribute(fundingSourceTbx, "aria-describedby");
+		return getText_custom(By.id(id));
+	}
+	
 	public void enterMaxGiftCardAmount(int i) {
 		clearMaxGiftCardTbx();
 		sendKeysUsingJavaScript(maxGiftCardAmtTbx, "document.getElementsByName('maximumGCAmount')[0].value="+i);
 	}
+	
+	public void clearFundingSource() {
+		clear_custom(fundingSourceTbx);
+	}
+
+	public String getToastMessage() {
```

Comment:
try to find other locator for Error message without contains text

Reply:
I am unable to locate the xpath of toast message because it appears for 1 second and disappears after 1 sec.
https://somup.com/c0iwo5zs4s